lib/screens/v2/localized_alert.ex

```diff
@@ -12,12 +12,11 @@ defmodule Screens.V2.WidgetInstance.Common.BaseAlert do
   alias Screens.V2.WidgetInstance.Alert, as: AlertWidget
   alias Screens.V2.WidgetInstance.{DupAlert, ElevatorStatus, ReconstructedAlert}
 
-  @type t ::
-          AlertWidget.t()
+  @type t :: AlertWidget.t()
           | DupAlert.t()
           | ReconstructedAlert.t()
           | ElevatorStatus.t()
-          | %{alert: Alert.t(), location_context: LocationContext.t()}
+          | %{alert: Alert.t(), location_context: LocationContext.t(), screen?: Screen.t()}
```

Comment:
Very small thing: `screen?:` as a key means it literally has a key named `:screen?` (since elixir atoms can have question marks). If you want to mark the field as optional like the `?` achieves in TypeScript, it's this much less elegant syntax:
```suggestion
          | %{optional(:screen) => Screen.t(), alert: Alert.t(), location_context: LocationContext.t()}
```
(It needs to come first because reasons)

Reply:
Oh my gosh 😅 hahahhaa I actually laughed aloud at this. My "codeswitching" failed miserably. So glad you noticed